### ISSUE TITLE
Deployment fails

### ISSUE THREAD

**Issue by hffmnn** [aws-quickstart/saas-identity-cognito#41]
Hi and thanks for this quickstarter.

When trying to deploy to eu-west-1, I get the following error after ~7 minutes:

```
Embedded stack arn:aws:cloudformation:eu-west-1:345384422074:stack/SaaS-identity-with-Cognito-IdentityStack-16N5Y0W3S8ZTI/12825580-3169-11ea-89fb-06135fdfc858 was not successfully created: The following resource(s) failed to create: [ECS, API].
```

Looking at the region table (as suggested in the deployment info), it looks like Amazon Cognito, AWS CodeBuild, Amazon, ECS, and Amazon ECR are supported.

I have a fresh account (created ~1 month ago, free tier eligible). Any ideas why this might happen?

---
EDIT 1: Some more information

AvailabilityZones | eu-west-1a,eu-west-1b | -
-- | -- | --
Company | somecompany | -
Email | somecompany@gmail.com | -
Firstname | some | -
InstanceType | t2.micro | -
KeyPairName | saas-identity-cognito | -
Lastname | name | -
OrderTable | Order | -
PrivateSubnet1CIDR | 10.0.0.0/19 | -
PrivateSubnet2CIDR | 10.0.32.0/19 | -
ProductTable | Product | -
PublicSubnet1CIDR | 10.0.128.0/20 | -
PublicSubnet2CIDR | 10.0.144.0/20 | -
QSS3BucketName | aws-quickstart | -
QSS3KeyPrefix | saas-identity-cognito/ | -
TenantTable | Tenant | -
UserTable | User | -
VPCCIDR | 10.0.0.0/16 | -

---

EDIT2: Deployment also fails in the default Oregon region:
```
Embedded stack arn:aws:cloudformation:us-west-2:345384422074:stack/SaaS-identity-with-Cognito-IdentityStack-1VWAIRCMN714P/93b84ee0-316c-11ea-ac44-0621d00a30ea was not successfully created: The following resource(s) failed to create: [ECS, API].
```

**Comment by asimSiddiquiSeer** [comment]
I think this is the same underlying issue as the bug I submitted yesterday. The top level error is identical. If you are able to look at the nested cloud formation tasks, it is going to be the nodeJs version. Since the rollback gets initiated as soon as it fails, you need to look at the events before the nested task is deleted.

**Comment by hffmnn** [comment]
Should be fixed with 73e8fce540e2709fb87c549c159290195d71ddba
